src/TableColumn/Tooltip.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace atk4\ui\TableColumn;
+
+use atk4\data\Field;
+
+/**
+ * Class Tooltip
+ *
+ * column to add a little icon to show on hover a text
+ * text is taken by the Row Model in $tooltip_field
+ *
+ * @usage   : $crud->addDecorator('paid_date',  new \atk4\ui\TableColumn\Tooltip('note'));
+ *
+ * @usage   : $crud->addDecorator('paid_date',  new \atk4\ui\TableColumn\Tooltip('note','error red'));
+ *
+ * @package atk4\ui\TableColumn
+ */
+class Tooltip extends Generic
+{
+    /**
+     *
+     * @var string $field_tooltip
+     */
+    public $icon = [];
```

Comment:
why empty array here if it's a string type

Reply:
was a typo